web/src/components/vis/HeatmapTile.vue

```diff
@@ -118,19 +127,41 @@ vis-tile-large(v-if="plot", title="Heatmap", expanded, download, :download-impl=
     sample-colorer(:dataset="dataset", v-model="sampleColor")
 
     v-toolbar.darken-3(color="primary", dark, flat, dense)
-      v-toolbar-title Dendogram
+      v-toolbar-title.switch-title Dendrogram
```

Comment:
Should the `.switch-title` class be removed?

Reply:
See 0030e51.